processing/src/main/java/org/apache/druid/query/operator/window/ranking/WindowRowNumberProcessor.java

```diff
@@ -137,4 +138,23 @@ public List<String> getOutputColumnNames()
   {
     return Collections.singletonList(outputColumn);
   }
+
+  @Override
+  public boolean equals(Object o)
```

Comment:
Do we need to add these functions anymore?

Reply:
Yes, I had to add this for the test added in MSQWindowTest.